<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	border: 1px solid #C07F00;
	background: #FFF7D4;
	border-radius: 6px;

}

QPushButton:hover {
	border: 1px solid black;
	background-color: #FFD95A;
}
QLineEdit {
	border: 1px solid #C07F00;
	border-radius: 6px;	
	padding: 6px;
	background: white;
}

QLineEdit:focus {
	border: 1px solid #F8AD25;
}

QLineEdit:hover {
	background-color: #FFD95A;
}

QComboBox {
	border: 0.5px solid #C07F00;
}

QComboBox:hover {
	background-color: #FFD95A;
}
</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>290</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BPG Nino Mtavruli</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>ექიმები</string>
    </attribute>
    <widget class="QTableWidget" name="doctors">
     <property name="geometry">
      <rect>
       <x>279</x>
       <y>0</y>
       <width>440</width>
       <height>260</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BPG Nino Mtavruli</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <column>
      <property name="text">
       <string>სახელი, გვარი</string>
      </property>
      <property name="font">
       <font>
        <family>BPG Nino Mtavruli</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ტელეფონი</string>
      </property>
      <property name="font">
       <font>
        <family>BPG Nino Mtavruli</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>კატეგორია</string>
      </property>
      <property name="font">
       <font>
        <family>BPG Nino Mtavruli</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="add_doctor_button">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>193</y>
       <width>220</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BPG Nino Mtavruli</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ექიმის დამატება</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="doc_fname_lname">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>75</y>
       <width>220</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BPG Nino Mtavruli</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>სახელი, გვარი</string>
     </property>
    </widget>
    <widget class="QComboBox" name="doc_category">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>145</y>
       <width>220</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BPG Nino Mtavruli</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>კოსმეტოლოგია</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ლაზერი</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="doc_title">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>30</y>
       <width>150</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BPG Nino Mtavruli</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ექიმის დამატება</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="doc_phone">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>110</y>
       <width>220</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BPG Nino Mtavruli</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>ტელეფონი</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>ზონები</string>
    </attribute>
    <widget class="QPushButton" name="add_zone_button">
     <property name="geometry">
      <rect>
       <x>49</x>
       <y>160</y>
       <width>220</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BPG Nino Mtavruli</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ზონის დამატება</string>
     </property>
    </widget>
    <widget class="QLabel" name="zone_title">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>49</y>
       <width>146</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BPG Nino Mtavruli</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ზონის დამატება</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="zone_name">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>120</y>
       <width>280</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BPG Nino Mtavruli</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>ზონა</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="zones">
     <property name="geometry">
      <rect>
       <x>319</x>
       <y>0</y>
       <width>400</width>
       <height>260</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BPG Nino Mtavruli</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <column>
      <property name="text">
       <string>ზონა</string>
      </property>
      <property name="font">
       <font>
        <family>BPG Nino Mtavruli</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>პროცედურები</string>
    </attribute>
    <widget class="QPushButton" name="add_procedure_button">
     <property name="geometry">
      <rect>
       <x>49</x>
       <y>160</y>
       <width>220</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BPG Nino Mtavruli</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>პროცედურის დამატება</string>
     </property>
    </widget>
    <widget class="QLabel" name="procedure_title">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>49</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BPG Nino Mtavruli</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>პროცედურის დამატება</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="procedure_name">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>120</y>
       <width>280</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BPG Nino Mtavruli</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>პროცედურა</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="procedures">
     <property name="geometry">
      <rect>
       <x>319</x>
       <y>0</y>
       <width>400</width>
       <height>260</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BPG Nino Mtavruli</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <column>
      <property name="text">
       <string>პროცედურა</string>
      </property>
      <property name="font">
       <font>
        <family>BPG Nino Mtavruli</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>doc_fname_lname</tabstop>
  <tabstop>doc_phone</tabstop>
  <tabstop>doc_category</tabstop>
  <tabstop>add_doctor_button</tabstop>
  <tabstop>doctors</tabstop>
  <tabstop>zone_name</tabstop>
  <tabstop>add_zone_button</tabstop>
  <tabstop>zones</tabstop>
  <tabstop>procedure_name</tabstop>
  <tabstop>add_procedure_button</tabstop>
  <tabstop>procedures</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
